.github/workflows/test.yml

```diff
@@ -3,9 +3,11 @@ on:
   push:
     branches-ignore: [main]
   workflow_dispatch:
+  pull_request:
 
 env:
   UT_DISABLE_NODE_CURRENT: true
+  NODE_VERSION_OVERRIDE: [lts/-1, lts/*]
```

Comment:
Do we need the `NODE_VERSION_OVERRIDE`? I thought this was already the criteria for the node version.

Reply:
I think this can be removed

Reply:
Yeas I was just experimenting here. After adding `UT_DISABLE_NODE_CURRENT: true` for some reason PRs that are from forks are still failing due to wrong node version (see [here](https://github.com/salesforcecli/plugin-lightning-dev/actions/runs/9702921966/job/26780247716)) but PRs from branches on the main repo pass (see [here](https://github.com/salesforcecli/plugin-lightning-dev/pull/73))

This didn't make sense to me and I was just curious to see if by adding `NODE_VERSION_OVERRIDE` I can fix the issue.